extensions/build.gradle.kts

```diff
@@ -0,0 +1,119 @@
+import com.vanniktech.maven.publish.SonatypeHost
+import java.net.URI
+
+plugins {
+    id("java-library")
+    id("signing")
+    id("com.github.johnrengelman.shadow") version "7.1.1"
+    id("com.vanniktech.maven.publish")
+    jacoco
+    id("com.google.protobuf") version "0.9.4"
+}
+
+repositories {
+    mavenCentral()
+}
+
+val jvmVersion = extra["jvmVersion"] as String
+val specsVersion = extra["specsVersion"] as String
+
+val javadocConfig by configurations.creating {
+    extendsFrom(configurations.testImplementation.get())
+}
+
+dependencies {
+    implementation(project(":core"))
+
+    // Use JUnit test framework.
+    testImplementation(libs.junit)
+
+    implementation("com.google.protobuf:protobuf-java:4.27.2")
```

Comment:
another set of version numbers

Reply:
I added a versions catalog and put it there